www/src/components/navigation/moreMenu.astro

```diff
@@ -1,85 +1,75 @@
 ---
 import * as CONFIG from "../../config";
 import ThemeToggleButton from "./ThemeToggleButton";
-//import ThemeToggleButton from "./ThemeToggleButton";
 
 export interface Props {
   editHref: string;
 }
 
 const { editHref } = Astro.props;
-const showMoreSection = CONFIG.COMMUNITY_INVITE_URL;
 ---
 
 <div class="flex flex-col w-full items-center sm:items-start pt-8">
-  {
```

Comment:
again, don't need to conditionally render these since we have the typesafety in place to warn if we were to remove these config options